frontend/static/js/process-editor.js

```diff
@@ -38,10 +38,6 @@ async function getFeatures() {
         .then(data => {
             let features = data['features'];
 
-            //TODO: auskommentieren
```

Comment:
Was ist mit dem ToDo, sehe das nicht in Issue? Hat sich das erledigt?

Reply:
das was auskommentiert werden soll war eine zeile drunter auskommentiert und wurde nach absprache mit tom entsprechend einfach gelöscht :)